### ISSUE TITLE
apply this method to data limited field.

### ISSUE THREAD

**Issue by lu77777777** [zhengzangw/PCS-FUDA#8]
Dear Zhengzang,

Thanks a lot. I tried to apply this method in a data-limited field. Last time, as you suggested, I employed the Cross-Entropy loss as a pretrain result. Then using this method. But the odd thing is the performance is even worse than before.

Do you know how to solve it? Thanks a lot!

Best regards,
Lu

**Comment by zhengzangw** [comment]
Sorry, but I do not know your specific application field. Maybe additional modifications should be made to the original methods. You can check the centroids of classes (or the tsne visualization) after pertaining to see whether they are well-separated.

**Comment by lu77777777** [comment]
Hi Zhengzang,

Thank you for your suggestions. I really appreciate it. Hopefully one day I can figure it out and if it is possible, we can work together.
